frontend/src/components/ui/flash-messages/flash-message.tsx

```
@@ -0,0 +1,89 @@
+      id={`flash-message-${id}`}
+      data-testid="flash-message"
+      data-type={type}
+    >
+      <SvgIcon
+        svg={icon}
+        className={buildClassNames('me-2 mt-[3px]', iconClasses)}
+      />
+      {hasHtml ? (
+        <div
+          className="flex-grow leading-5"
+          dangerouslySetInnerHTML={{
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)